src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/StringSearchParam.sql

```diff
@@ -4,6 +4,9 @@
     ResourceSurrogateId         bigint              NOT NULL,
     SearchParamId smallint NOT NULL,
     Text nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL,
+    CONSTRAINT PK_StringSearchParam PRIMARY KEY NONCLUSTERED (ResourceTypeId, SearchParamId, Text, ResourceSurrogateId)
```

Comment:
Is this guaranteed to be unique?  What about TextOverflow?

Reply:
This is a good point. Somehow I missed to test the case where Text would only differ in last few bytes and thus multiple rows could have same Text but TextOverflow would be different. Since TextOverflow datatype is varchar(max), ideally this should not considered to be part of primary key so more analysis is required to find a primary key for this table. 
Created [user story](https://microsofthealth.visualstudio.com/Health/_boards/board/t/Olympus/Stories/?workitem=87237) to investigate if we can add isTextOverflowDiff as bit which will only be set to 1 if TextOverflow is not null and different for two rows without impacting the current fhir functionality.